packages/components/popconfirm/src/popconfirm.vue

```diff
@@ -92,19 +61,21 @@ export default defineComponent({
       visible.value = false
       emit('cancel')
     }
-    const confirmButtonText_ = computed(() => {
-      return props.confirmButtonText || t('el.popconfirm.confirmButtonText')
-    })
-    const cancelButtonText_ = computed(() => {
-      return props.cancelButtonText || t('el.popconfirm.cancelButtonText')
-    })
+    const confirmButtonTextFinal = computed(
+      () => props.confirmButtonText || t('el.popconfirm.confirmButtonText')
+    )
+    const cancelButtonTextFinal = computed(
+      () => props.cancelButtonText || t('el.popconfirm.cancelButtonText')
+    )
```

Comment:
Can we use `_confirmButtonText` because final here seems weird and `_var`  indicates internal variable which in this case would be more appropriate.